cypress/integration/untitled.spec.js

```diff
@@ -0,0 +1,6 @@
+// untitled.spec.js created with Cypress
```

Comment:
Please add correct name for test or delete this test

Reply:
Done